var/spack/repos/builtin/packages/fujitsu-mpi/package.py

```diff
@@ -27,6 +28,24 @@ def install(self, spec, prefix):
         raise InstallError(
             'Fujitsu MPI is not installable; it is vendor supplied')
 
+    @property
+    def headers(self):
+        hdrs = HeaderList(find(self.prefix.include, 'mpi.h', recursive=True))
```

Comment:
```suggestion
        hdrs = find_headers('mpi', self.prefix.include, recursive=True)
```

Reply:
@adamjstewart Thanks for review. I fixed it.